pallets/anchors/src/lib.rs

```diff
@@ -516,15 +533,15 @@ impl<T: Config> Pallet<T> {
 			.take(MAX_LOOP_IN_TX as usize)
 			// get eviction date of the anchor given by index
 			.filter_map(|idx| {
-				let anchor_index = <AnchorIndexes<T>>::get(idx)?;
-				let eviction_date = <AnchorEvictDates<T>>::get(anchor_index).unwrap_or_default();
-				Some((idx, anchor_index, eviction_date))
+				let anchor_id = <AnchorIndexes<T>>::get(idx)?;
```

Comment:
I have taken the liberty of renaming this variable because it refers to the id of the anchor, not the index.

Reply:
Nice!